docs/conf/australia/2021/sponsors/prospectus.rst

```diff
@@ -41,32 +41,42 @@ Sponsorship Packages
 
 The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
 
+First Draft
+------------
+
+The **First Draft** package is only available to startups (under 15 employees), non-profits, and open source organizations.
+
+- **Increased** Two (2) tickets_
+- **Increased** One (1) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A short description (250 characters) and logo of your company on the conference website
+- Name included in welcome announcement in email newsletters and social media
+
+The **First Draft** package costs AUD **{{sponsorship.first_draft.price}}**.
```

Comment:
We should probably just put the currency in the `sponsorship.first_draft.price` -- so it can be shown everywhere we display it, if needed.